easybuild/easyconfigs/m/magma/magma-1.6.2-CrayGNU-5.2.40.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "ConfigureMake"
+
+name = 'magma'
+version = '1.6.2'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = "The MAGMA project aims to develop a dense linear algebra library similar to LAPACK but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."
+
+toolchain = {'name': 'CrayGNU', 'version': '5.2.40'}
+toolchainopts = {'pic': True, 'dynamic': True,  'openmp': True, }
+
+preinstallopts = "export EBINSTALLPREFIX=%(installdir)s && "
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+
+skipsteps = ['configure']
+
+builddependencies = [ ('cudatoolkit/6.5.14-1.0502.9613.6.1', EXTERNAL_MODULE) ]
+ 
+start_dir =  'magma-1.6.2/' 
```

Comment:
Tarball contains only one directory, it is not clear for me why I need that...
Here's what I get if I comment out start_dir:
https://gist.github.com/gppezzi/fba34e0dfcb6f78589a9